### ISSUE TITLE
Not respecting the line breaks

### ISSUE THREAD

**Issue by ph7jack** [heybourn/headwind#191]
**Describe the bug**
The sorting is not respecting the line breaks to make it easy to read

**Before**
![image](https://user-images.githubusercontent.com/38913462/182393650-8c2aeb3d-8890-4e8c-99c9-fc6ae080fb4d.png)

**After**
![image](https://user-images.githubusercontent.com/38913462/182393696-f30e024c-5f4a-4bfe-9611-4e4e0b3a6535.png)

**Comment by hacknug** [comment]
I don't think this is something headwind supports by default. Maybe you can change the regex so it treats each line as an independent group to sort?
